couchpotato/core/media/movie/providers/info/couchpotatoapi.py

```diff
@@ -102,12 +106,141 @@ def getInfo(self, identifier = None, adding = False, **kwargs):
 
         return {}
 
+
     def getReleaseDate(self, identifier = None):
         if identifier is None: return {}
 
         dates = self.getJsonData(self.urls['eta'] % identifier, headers = self.getRequestHeaders())
-        log.debug('Found ETA for %s: %s', (identifier, dates))
 
+        #This grabs release date info from omdbapi/rottentomatoes
+        temp2 = self.getJsonData("http://www.omdbapi.com/?i=%s&tomatoes=true&plot=short&r=json" % identifier)
+        title = temp2['Title']
+        year = int(temp2['Year'])
+
+        ddate=0 #throw away what couchpotatoai is returning since it is garbage at this time
+        tdate=0
+        dvd_date= temp2['DVD']
+        theater_date=temp2['Released']
+        if theater_date != 'N/A' and year >1972:
+            p='%d %b %Y'
+            tdate=int(time.mktime(time.strptime(theater_date,p)))
+
+        if dvd_date != 'N/A' and year >1972:
+            p='%d %b %Y'
+            ddate=int(time.mktime(time.strptime(dvd_date,p)))
+           
+        if (ddate !=0):    
+            if ddate < tdate:
+                ddate = 0 #if the dvd release date occurs before the theater release date, assume the data is wrong
+                tdate = 0
+
+        if (ddate or tdate):
```

Comment:
I only noticed this because I have found a release where TMDb DVD release date is present, but the OMDBAPI was not. Personally I only really care about the DVD ETA, so I thought it would be good that both services are checked. The way it was TMDb would probably never be checked for a DVD release date.

In all honesty I was only concerned with your great ETA gathering/display changes and don't have a Netflix account so I didn't pick up any issues there.

Thanks for your work by the way, it's good to be able to see in one place what stages each movie is at.